src/storage/RustSdkCryptoStorageProvider.ts

```diff
@@ -4,23 +4,29 @@ import * as mkdirp from "mkdirp";
 import * as path from "path";
 import * as sha512 from "hash.js/lib/hash/sha/512";
 import * as sha256 from "hash.js/lib/hash/sha/256";
+import { StoreType as RustSdkCryptoStoreType } from "@matrix-org/matrix-sdk-crypto-nodejs";
 
 import { ICryptoStorageProvider } from "./ICryptoStorageProvider";
 import { IAppserviceCryptoStorageProvider } from "./IAppserviceStorageProvider";
 import { ICryptoRoomInformation } from "../e2ee/ICryptoRoomInformation";
 
+export { RustSdkCryptoStoreType };
+
 /**
- * A crypto storage provider for the default rust-sdk store (sled, file-based).
+ * A crypto storage provider for the file-based rust-sdk store.
  * @category Storage providers
  */
 export class RustSdkCryptoStorageProvider implements ICryptoStorageProvider {
```

Comment:
I do wonder if we should have subclassed these to RustSdkCryptoStoreSled and `RustSdkCryptoStoreSqlite` so the type was implicit.

Reply:
IMO we're limited by how the rust-sdk defines these, which also uses one class + a parameter for Sled/SQLite/other future store types.